backend/people/migrations/0020_fill_workshops.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.26 on 2020-10-22 19:47
+from __future__ import unicode_literals
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    Workshop = apps.get_model('events', 'Workshop')
+    WSS = apps.get_model('WSS', 'WSS')
+    Speaker = apps.get_model('people', 'Speaker')
+
+    wss = WSS.objects.get(year=2020)
+
+    Workshop.objects.create(
+        wss=wss,
+        title='Conceptual Blending in Computational Creativity',
+        syllabus='Modeling humans\' cognitive processes in a computable way is our goal. While machines communicate with humans, they need to understand complicated products produced by human’s creativity. Human creativity appears in music, art, jokes, novel stories, movies, and every other area. My current focus is on conceptual blending. Conceptual blending (CB) is a vital cognitive mechanism by which two or more mental spaces are integrated to produce new concepts (Fauconnier and Turner 2002), e.g., when a house and boat blend and a house-bout is created. Visual Metaphor is a similar area. Despite not all visual blends being visual metaphors, most research is conducted about advertisements and focuses on them. Creative systems can understand human creativity, produce innovative products, and enhance human creativity (The International Association for Computational Creativity (ACC)). I wish to explain this fantastic area in detail.',
+        speaker=Speaker.objects.filter(name='Maryam Sadat Eslami').last()
+    )
```

Comment:
Please set the `polymorphic_ctype` for these objects.

Reply:
Done